guides/source/action_view_overview.md

```diff
@@ -234,23 +222,26 @@ call to `<%= render "ad_banner" %>` by first searching for
 `app/views/products/_ad_banner.html.erb` before falling back to
 `app/views/application/_ad_banner.html.erb`.
 
-### `render` with `locals` Option
+### Passing Data to Partials With `locals` Option
 
-When rendering a partial, each key in the `locals:` option is available as a
-partial-local variable:
+When rendering a partial, we can pass data to the partial from the rendering view. We use the `locals:` options hash for this. Each key in the `locals:` option is available as a partial-local variable:
```

Comment:
With phrases like 'rendering view' and 'partial-local variable' this section really made me stumble. 

I also easily overlooked that `locals` is a hash, so can we add another key to the examples to show that using `locals` gives you flexibility? Maybe price and/or category?

Suggestion (or something like): 
When rendering a partial, we can pass data to the partial from the rendering view. We use the `locals:` options hash for this. Each key in the `locals:` option is available as a partial-local variable, in other words: a variable that is only accessed when that specific partial is being called.

In the above example, my_product is the local variable available in the partial. It was assigned the value of @product in the original view. 

This allows you to reuse the _product.html.erb partial across different views while customizing the product data displayed each time. 

If a template refers to a variable that isn't passed into the view as part of the locals: option, the template will raise an ActionView::Template::Error. A partial-local variable only works in the context of the partial in which it is defined.